news/2021-06-14-mca-ayim-2019.md

```diff
@@ -0,0 +1,55 @@
+
+![](/wiki/shared/news/2021-06-13-mca-ayim-2019/banner.png)
+
+It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later on in the year.
+
+With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.
+
+
+## Schedule for MCA and AYIM 2019
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
+| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
+| Results Stream | 2021-07-10 / 2021-07-11 |
+| Results posted to Website | 2021-07-12 00:00 |
+
+## A Year In Mapping 2019
+
+AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and many, many more records than you can think of.
+
+**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to *everyone*. Know some banger maps from 2019 or a mapper who left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open for everyone to participate in, so let your positivity be heard!
+
+Writing comments will be open **until** the results of MCA are posted.
+
+All of these statistics and features can be found on the [AYIM Website](https://ayim.corsace.io)
+
+## Mapper's Choice Awards 2019
+
+MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards throughout multiple different categories. 
+
+Eligible users are those who ranked a beatmap, guest difficulty, or storyboard in 2019.
+
+MCA has three stages: Nomination, voting, and results.
+
+- **Nomination:** Eligible users can submit any maps/storyboards or users from 2019 they feel are deserving of recognition in their respective categories.
+- **Voting:** Eligible users can vote on the pool of submitted nominations.
+- **Results:** Will be based off the results of voting and feature a livestream discussing each of the top winners. 
+
+Those who win regular awards on their maps or storyboards in the results will be given 2 months of supporter, and grand prize winners will receive 6 months of supporter.
+
+Overall, this event will give perspective on what the community of that time finds to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019! 
```

Comment:
```suggestion
Overall, this event will give perspective on what the community of that time found to be the best maps of the year and will serve as an awesome reminder of what we've seen from 2019! 
```

Reply:
This event is being hosted _now_ reflecting on 2019, so it'd be present tense. People's impressions and what they remember most inevitably changes over time and results now would not be the same as then.